question21/main.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+const wrapper = document.getElementById('js-wrapper');
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
+
+const showLoadImg = () => {
+  const loadImg = createElementWithClassName("img", "loading");
+  loadImg.src = "./img/loading-circle.gif";
+  loadImg.id = 'js-loading';
+  wrapper.appendChild(loadImg);
+} 
+
+const removeLoadImg = () => {
+  document.getElementById('js-loading').remove();
+} 
+
+const getData = new Promise((resolve) => {
+  showLoadImg();
+  setTimeout(() => {
+    resolve(fetch('https://mocki.io/v1/5381547b-04ff-4a89-b169-dd84425f5196'));
+  }, 3000);
+})
+
+//Get json data
+const callApi = async() => {
+  try {
+    const response = await getData;
+    if (!response.ok) {
+      console.error(`${response.status}:${response.statusText}`);
+    }
+    const json = await response.json();
+    return json.data;
+  }
+  catch(error) {
+    console.error(error);
+  }
+  finally {
+    removeLoadImg();
+  }
+}
+
+const init = async() => {
+  const userData = await callApi();
+  if (userData){
+    renderTableElement(userData);
+    } else {
+    wrapper.textContent = "データの表示に失敗しました";
+    console.error('データの表示に失敗しました'); 
```

Comment:
nits
The error text might be better in English.
Mr Moriken's comment.
https://discord.com/channels/726252977928667179/818786882711322624/982420899376558163

Reply:
Oh I totally missed it!! Thank you for pointing it out! Fixed so please confirm.
fixed commit : https://github.com/csakyo/homework/pull/36/commits/57897a150a121d3d923342b7d75ae2bbf9786768